panel/src/components/Forms/Field/EntriesField.vue

```diff
@@ -0,0 +1,438 @@
+					@click="$refs.options.toggle()"
+				/>
+				<k-dropdown-content ref="options" :options="options" align-x="end" />
+			</k-button-group>
+		</template>
+	</k-field>
+</template>
+
+<script>
+import { props as Field } from "../Field.vue";
+import { props as Input } from "../Input.vue";
+
+export default {
+	mixins: [Field, Input],
+	inheritAttrs: false,
+	props: {
+		/**
+		 * The text, that is shown when the field has no entries.
+		 */
+		empty: String,
+		/**
+		 * Field attrs for the form
+		 */
+		field: {
+			type: Object,
+			default: () => {}
+		},
+		/**
+		 * Upper limit of entries allowed
+		 */
+		max: Number,
+		/**
+		 * Lower limit of entries required
+		 */
+		min: Number,
+		/**
+		 * Whether to allow sorting of entries
+		 */
+		sortable: {
+			type: Boolean,
+			default: true
+		},
+		value: {
+			type: Array,
+			default: () => []
+		}
+	},
+	emits: ["input", "sort"],
+	data() {
+		return {
+			entries: []
+		};
+	},
+	computed: {
+		/**
+		 * Returns draggable options
+		 * @returns {object}
+		 */
+		dragOptions() {
+			return {
+				disabled: this.isSortable === false,
+				handle: true,
+				list: this.entries,
+				sort: this.isSortable
+			};
+		},
+		/**
+		 * Returns whether the entries can be sorted
+		 * @returns {boolean}
+		 */
+		isSortable() {
+			if (this.disabled === true) {
+				return false;
+			}
+
+			if (this.entries.length <= 1) {
+				return false;
+			}
+
+			if (this.sortable === false) {
+				return false;
+			}
+
+			return true;
+		},
+		/**
+		 * Returns if new entries can be added
+		 * @returns {boolean}
+		 */
+		more() {
+			if (this.disabled === true) {
+				return false;
+			}
+
+			if (this.max && this.entries.length >= this.max) {
+				return false;
+			}
+
+			return true;
+		},
+		options() {
+			return [
+				{
+					disabled: this.entries.length === 0 || this.disabled,
+					icon: "template",
+					text: this.$t("copy.all"),
+					click: this.copyAll
+				},
+				"-",
+				{
+					disabled: this.entries.length === 0 || this.disabled,
+					icon: "trash",
+					text: this.$t("delete.all"),
+					click: this.removeAll
+				}
+			];
+		},
+		values() {
+			return this.entries.map((entry) => entry.value);
+		}
+	},
+	watch: {
+		value: {
+			handler(entries) {
+				// no need to add ids again if the values are the same
+				if (entries === this.values) {
+					return;
+				}
+
+				this.entries = entries.map((value) => {
+					return {
+						id: this.$helper.uuid(),
+						value
+					};
+				});
+			},
+			immediate: true
+		}
+	},
+	methods: {
+		add(index = null, value = null) {
+			if (this.more === false || this.disabled === true) {
+				return;
+			}
+
+			const entry = {
+				id: this.$helper.uuid(),
+				value: value ?? ""
+			};
+
+			if (index !== null) {
+				this.entries.splice(index, 0, entry);
+			} else {
+				this.entries.push(entry);
+				index = this.entries.length - 1;
+			}
+
+			this.save();
+
+			this.$nextTick(() => {
+				this.focus(index);
+			});
```

Comment:
```suggestion
			await this.$nextTick();
			this.focus(index);
```

(but then the method needs to become `async` too).